Anlab.Core/Models/OrderDetails.cs

```diff
@@ -23,6 +23,11 @@ public class OrderDetails
 
         public string LabComments { get; set; }
         public decimal AdjustmentAmount { get; set; }
+        public decimal GrandTotal { get

```

Comment:
@laholstege Lets double check with @srkirkland about this one.
I don't think it is needed because it is getting added in the PopulateOrderWithLabDetails method already. But this method would need to be called. It doesn't need to be called in the save method anymore because that is just for the react form.